src/styles/form.scss

```diff
@@ -0,0 +1,51 @@
+.form {
+  width: 40%;
+
+  @media (max-width: $displayLaptop) {
+    margin-top: 32px;
+    width: auto;
+  }
+
+  &__input {
+    resize: vertical;
+    display: block;
+    width: 100%;
```

Comment:
so block or 100% width?

Reply:
block and width. it is input, it is unpredictable https://prnt.sc/thk3hy